docs/gno-tooling/cli/gnodev.md

```diff
@@ -35,15 +36,45 @@ For hot reloading, `gnodev` watches the examples folder, as well as any specifie
 gnodev ./myrealm
 ```
 
+## Keybase and Balance
+
+Gnodev will, by default, load your keybase located in the GNOHOME directory, giving all your keys (almost) unlimited fund. 
+
+All realms will be by the `-genesis-creator`, but You can also pass query options to the realm path to load a
+realm with specific creator and deposit:
+```
+gnodev ./myrealm?creator=foo&deposit=42ugnot``
+```
+
+### Additional Account
+Use the `-add-account` flag with the format `<bech32/name>[:<amount>]` to add a specific address or key name
+from your local keybase. You can set an optional amount for this address. Repeat this command to add multiple
```

Comment:
With `--add-account`, you can specify or add an account with a specific amount. It's handy when you're on a remote machine without a local keybase. You can add one or multiple addresses using `--add-account`. If the address is already in the local keybase, you can use this flag set a specific amount on it the address (instead of an unlimited one)